### ISSUE TITLE
conv layer elements nan

### ISSUE THREAD

**Issue by mtxing69** [rbgirshick/py-faster-rcnn#885]
Hi everyone，
I downloaded the pascal_voc dataset, and got only person and dog these two classes out to try to train a simplified net.  I tried to train from the start, so I didn`t use any pretrained models and initialized the conv layers using 'xavier'. Then the trouble came as follows:
1. It didn`t converge: the train loss swung between 0.8 and 1.2.
2. I print the layers elements out : the conv layers are all almost nan and the BatchNorm layers are all almost zero.
Anyone has any ideas ?
![image](https://user-images.githubusercontent.com/20680261/52924868-8f48b400-3369-11e9-90e2-bd24de85f532.png)
